projects/packages/waf/changelog/add-waf-stats-class

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: added
+
+Added a Waf_Stats class
```

Comment:
```suggestion
Added a Waf_Stats class
Add statistics to the firewall settings screen
```
I'm going to squash all of the changelogs in the `add/protect-waf-phase-3` branch, but just because it is interesting:

I recently learned it is recommended to write our changelog content in future tense (i.e. "Add a Waf_Stats class"). Someone from crew or the release team told me this... that said I can't find the slack conversation or anything to back this up.